Options.cmake

```diff
@@ -92,22 +100,13 @@ IF (MSVC OR MINGW)
 #
    option (DEBUG_BREAK_AT_FATAL_SIGNAL
        "Enable Visual Studio break point when receiving a fatal exception. In __DEBUG mode only" OFF)
-    IF(DEBUG_BREAK_AT_FATAL_SIGNAL)
-      LIST(APPEND G3_DEFINITIONS DEBUG_BREAK_AT_FATAL_SIGNAL)
-      MESSAGE("-DDEBUG_BREAK_AT_FATAL_SIGNAL=ON                  Break point for fatal signal is enabled for __DEBUG.") 
-    ELSE() 
-       MESSAGE("-DDEBUG_BREAK_AT_FATAL_SIGNAL=OFF\t\t\tBreak point for fatal signal is disabled") 
-    ENDIF(DEBUG_BREAK_AT_FATAL_SIGNAL)
-
-
-
-# Option for adding windows shared library
-    option (ADD_BUILD_WIN_SHARED  "Build shared library on Windows" OFF)
-       IF(ADD_BUILD_WIN_SHARED)
```

Comment:
I have added the G3_SHARED_LIB option as was proposed in another pull request. I can rename it to ADD_BUILD_WIN_SHARED but as we are not adding but it but rather build as either shared or static I thought the name more appropriate